fellowship.js

```diff
@@ -16,33 +16,107 @@ var fellowship = {
   lands: [
     "The Shire", "Rivendell", "Mordor"
   ],
+
   makeMiddleEarth: function(lands) {
-    console.log(lands)
     // create a section tag with an id of middle-earth
     // add each land as an article tag
     // inside each article tag include an h1 with the name of the land
     // append middle-earth to your document body
-  },
+    console.log(lands);
+    var middleEarth = document.createElement("section");
+    middleEarth.id = "middle-earth";
+    console.log(middleEarth);
+    for(var i = 0; i < lands.length; i++){
+      var articleM = document.createElement("article");
+      var heading = document.createElement('h1');
+      articleM.id = lands[i];
+      heading.textContent = lands[i];
+      articleM.appendChild(heading);
+      middleEarth.appendChild(articleM);
+  }
+    document.body.appendChild(middleEarth);
+},
+
   makeHobbits: function(hobbits) {
     // display an unordered list of hobbits in the shire
     // give each hobbit a class of hobbit
+    console.log(hobbits);
+    var hobbitUL = document.createElement('ul');
+    for(var i = 0; i < hobbits.length; i++){
+      var hobbitList= document.createElement('li');
+      hobbitList.className = "hobbit";
+      hobbitList.textContent = hobbits[i];
+      hobbitUL.appendChild(hobbitList);
+    }
+    var shire = document.getElementById('The Shire');
+    shire.appendChild(hobbitUL);
   },
   keepItSecretKeepItSafe: function() {
     // create a div with an id of 'the-ring'
     // add the ring as a child of Frodo
+    var theRing = document.createElement("div");
+    theRing.id = "the-ring";
+    theRing.textContent = "The Ring";
+    var frodo = document.getElementsByClassName('hobbit')[0];
+    frodo.appendChild(theRing);
   },
   makeBuddies: function(buddies) {
     // create an aside tag
     // display an unordered list of buddies in the aside
     // insert your aside before rivendell
+    console.log(buddies);
+    var aside = document.createElement("aside");
+    var buddiesUL = document.createElement("ul");
+    for (var i = 0; i < buddies.length; i++){
+    var buddiesList = document.createElement("li");
```

Comment:
don't forget to indent inside curly braces (i.e. inside of functions, for loops, if statements, etc)